safe_cli/operators/hw_accounts/ledger_account.py

```diff
@@ -0,0 +1,102 @@
+import warnings
+
+from eth_account.datastructures import SignedTransaction
+from eth_account.signers.base import BaseAccount
+from hexbytes import HexBytes
+from ledgerblue import Dongle
+from ledgereth import create_transaction, sign_typed_data_draft
+from web3 import Web3
+from web3.types import TxParams
+
+
+class LedgerAccount(BaseAccount):
+    def __init__(self, path, address, dongle: Dongle):
+        """
+        Initialize a new ledger account (no private key)
+
+        :param path: path derivation
+        :param ~eth_account.account.Account account: the key-unaware management API
+        """
+        self._address = address
+        self.path = path
+        self.dongle = dongle
+
+    @property
+    def address(self):
+        return self._address
+
+    @property
+    def privateKey(self):
+        """
+        .. CAUTION:: Deprecated for :meth:`~eth_account.signers.local.LocalAccount.key`.
+            This attribute will be removed in v0.5
+        """
+        warnings.warn(
+            "privateKey is deprecated in favor of key",
+            category=DeprecationWarning,
+        )
+        return None
```

Comment:
```suggestion
        return self.key
```

Reply:
we can't export ledger private key should keep on ledger device.

Reply:
It can be exported at all. But what I mean is that when a function is deprecated you should call the function you are deprecating it for, not adding the logic again